svgo/removeUnknownsAndDefaults.js

```diff
@@ -1,132 +1,225 @@
 /* eslint-disable */
```

Comment:
Last time I forgot to update this file when updating SVGO to the latest version and it caused the application to crash when processing SVGs.